pages/cart/cart.css

```diff
@@ -0,0 +1,200 @@
+@import "../../css/default.css";
+@import "../../css/navbar.css";
+@import "../../css/footer.css";
+
+.cart-section {
+  margin-top: 6rem;
+}
+
+.cart-container {
+  padding: 0 1rem;
+}
+
+.cart-container .grid {
+  gap: 1.5rem;
+}
+
+.card.cart-horizontal-card {
+  padding: 0.5rem;
+  border-radius: 4px;
+  margin-bottom: 1rem;
+  width: 100%;
+  overflow: hidden;
+  border: 1px solid var(--gray-300);
+}
+
+.card.cart-horizontal-card .card-body {
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+  height: 100px;
+  margin-bottom: 8px;
+}
+
+.card.cart-horizontal-card .card-body img {
+  width: 30%;
+}
+
+.card.cart-horizontal-card .card-title {
+  font-size: 1rem;
+}
+
+.card.cart-horizontal-card .card-footer {
+  display: flex;
+  gap: 10px;
+}
+
+.card.cart-horizontal-card .card-footer .btn {
+  width: 100%;
+}
+
+.cart-horizontal-card .card-price {
+  padding: 0.2rem 0;
+}
+
+.cart-horizontal-card .price-after-discount {
+  font-size: 0.85rem;
+}
+
+.cart-horizontal-card .card-price > :is(.price-before-discount, .discount) {
+  font-size: 0.65rem;
+}
+
+.select-quantity {
+  font-size: 0.85rem;
+  color: var(--gray-800);
+  margin-top: 8px;
+  width: fit-content;
+  background: var(--gray-50);
+  padding: 2px 6px;
+  border-radius: 4px;
+}
+
+.select-quantity select {
+  border: none;
+  outline: none;
```

Comment:
I think we should not remove outline property. 
For keyboard or other pointing devices, it might create problems

Correct me If I am wrong